src/api/spec/factories/kiwi_image.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :kiwi_image, class: Kiwi::Image do
+    name { Faker::Name.name }
+    md5_last_revision { Faker::Crypto.md5 }
+
+    factory :kiwi_image_with_package do
+      transient do
+        package_name { 'package_with_kiwi_image' }
+      end
+
+      after(:create) do |image, evaluator|
+        image.package = create(:package, name: evaluator.package_name)
```

Comment:
this two lines cna be changed by:

``` Ruby
create(:package, name: evaluator.package_name, kiwi_image: image)
```

as you don't need to save it if you write it like that